packages/tadviewer/src/actions.ts

```diff
@@ -77,7 +77,10 @@ export const setQueryView = async (
   const baseSchema = await aggtree.getBaseSchema(dsc, baseQuery);
 
   // start off with all columns displayed:
-  const displayColumns = baseSchema.columns.slice();
+  const lastColumn = appState.showRecordCount
+    ? baseSchema.columns.length
+    : baseSchema.columns.length - 1;
+  const displayColumns = baseSchema.columns.slice(0, lastColumn);
```

Comment:
This is a little bit brittle, because it implicitly depends on "Rec" being the last column added to the query (and schema).
"Rec" is added in two places: In `getBaseSchema` and `vpivot`, both in `aggtree/src/aggtree.ts`. 
Could you please take a look at adding `showRecordCount` as a parameter to those functions, and then plumbing that through so that it's fed from the `showRecordCount` in `AppState`?
That should cleanly remove "Rec" entirely from the schema and query when `showRecordCount` is `false`, which seems preferable to adding implicit assumptions that it will always be the last column.  I think if you did that, these lines won't be needed.